nb_conda_kernels/manager.py

```diff
@@ -16,6 +17,8 @@
 
 CACHE_TIMEOUT = 60
 
+CONDA_EXE = os.environ.get("CONDA_EXE", "conda")
```

Comment:
This is the only functional change in the code: using the `CONDA_EXE` variable, if it exists, to determine where `conda` is located. This is necessary for `conda 4.4+` where the conda application is no longer on the PATH.

Reply:
This is pulled directly from #79

Reply:
I wonder if you can leave this change out of this PR, to be discussed separately in #79. Your tests should all run fine without it (since they still use the pre-4.4 style installation with conda on the path see ../travis.yml:31 and ../appveyor.yml:55)